registry/static/css/award_document.css

```diff
@@ -0,0 +1,105 @@
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    background-color: grey; /* #FAFAFA */
+    font-family: "Arial","Helvetica Neue","Helvetica","sans-serif";
```

Comment:
Just a curious question - is some font from these installed on Linux systems by-default? But I guess, that the `"sans-serif"` option is enough.

Reply:
I guess it is not choosed by OS but by my plugin in codium (VScode text editor) which gives me IntellSense. Default mode gives you options as a set of a  several fonts. Some developers recommend to define several fonts to attribute font-family for the case when one or more of them wouldn't be available than it will choose next in the order. In our case Arial should work in 99% of cases so this point of view lost its argument value.

I suggest to use: 'font-family: "sans-serif", "Arial"'

The other side of coin is if we can use sans-serif as font to official document. I have no idea if exists some standard for that.

Reply:
`sans-serif` should be last and it shuldn't be in quotes; it's [a fallback that's always available](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name).